boltons/pathutils.py

```diff
@@ -0,0 +1,212 @@
+"""
+Functions for working with filesystem paths.
+
+The ``expandpath`` function expands the tilde to $HOME and environment variables
+to their values.
+
+The ``augpath`` function creates variants of an existing path without having to
+spend multiple lines of code spliting it up and stitching it back together.
+
+The ``shrinkuser`` function replaces your home directory with a tilde.
+
+The ``userhome`` function reports the home directory of the current user of the
+operating system.
+"""
+from __future__ import print_function
+
+from os.path import dirname
+from os.path import exists
+from os.path import expanduser
+from os.path import expandvars
+from os.path import join
+from os.path import normpath
+from os.path import split
+from os.path import splitext
```

Comment:
How about importing these in this style:

```python 
from os.path import (dirname, 
                     exists, 
                     expanduser, ...)`

Reply:
This is an artifact of my :AutoImport vim command, which automatically tries to add import statements for undefined names. I can refactor it to the concise style.

Reply:
This is fixed.